pkg/constants/constants.go

```diff
@@ -19,4 +19,12 @@ const (
 	StatusTriggered    = "triggered"
 	StatusAcknowledged = "acknowledged"
 	StatusResolved     = "resolved"
+
+	//ScheduleIDs for fetching oncalls
+	PrimaryScheduleID = "P995J2A"
```

Comment:
@MitaliBhalla  could you provide the pagerduty documentation link as a comment to IDs (where all the codes are listed with their meanings) for future reference.